pkg/operator/ceph/version/version.go

```diff
@@ -109,6 +136,26 @@ func ExtractCephVersion(src string) (*CephVersion, error) {
 	return &CephVersion{major, minor, extra}, nil
 }
 
+// ExtractCephCSIVersion extracts the major, minor and extra digit of a Ceph release
+func ExtractCephCSIVersion(src string) (*CephCSIVersion, error) {
+	m := versionCSIPattern.FindStringSubmatch(src)
+	if m == nil {
+		return nil, fmt.Errorf("failed to parse version from: %s", src)
+	}
+
+	major, err := strconv.Atoi(m[1])
```

Comment:
is it good to validate length of `m` before accessing its index?

Reply:
Good point!